canonical-web-and-design/ubuntu.com
# Rebuild openstack contact form

## Done
- Updated contact form on the following pages. 
  - [ ]  `/openstack#get-in-touch` 
  - [ ] `/openstack/what-is-openstack#get-in-touch`
  - [ ] `/openstack/features#get-in-touch`
  - [ ] `/openstack/consulting#get-in-touch`
  - [ ] `/openstack/compare#get-in-touch`
  - [ ] `/openstack/install#get-in-touch`
  - [ ] `/openstack/managed#get-in-touch` (The form is extended with the "Where do you want your OpenStack?" question on this page ) 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please check page against [copy doc](https://docs.google.com/document/d/140P1Z--hkWuW9-S2cNAR_uCW3BRW1VPKtFWkDtc7eBk/edit) and resolve comments


## Issue / Card

Fixes [#4382](https://github.com/canonical-web-and-design/web-squad/issues/4382)  [#4383](https://github.com/canonical-web-and-design/web-squad/issues/4383)  [#4384](https://github.com/canonical-web-and-design/web-squad/issues/4384)

## Screenshots

![image](https://user-images.githubusercontent.com/57550290/132016449-4e1d8b40-414b-4993-b3a6-4dd9dc71ebdb.png)
